theme/static/js/hs-vue/subject-keywords-app.js

```diff
@@ -92,6 +106,10 @@ let subjKeywordsApp = new Vue({
         safeJS: function (input) {
             input.replace("\\", "\\\\")
             return encodeURIComponent(input)
+        },
+        newKeywordsLength: function () {
+            let newVal =  this.resKeywords.join("") + this.newKeyword;
+            return newVal.length;
         }
```

Comment:
```suggestion
        newKeywordsLength: function () {
            let newVal =  this.resKeywords.join(",") + this.newKeyword;
            return newVal.length;
        }
```
☝️ perhaps this will give you a more accurate count of the KW length?

Reply:
Looks to me like the SubjectsForm is enforcing 500chars _including commas_:
https://github.com/hydroshare/hydroshare/blob/2.15.10/hs_core/forms.py#L670

Reply:
Okay let's look at a demo just addressing the frontend changes you're making in [subject-keywords-app.js](https://github.com/hydroshare/hydroshare/pull/5687/files/cbcc7bfe2bd50cf02cf4a9e65da70210a7fd9384#diff-139c5573aa825c90dfa543211c94ba22598083ace44ef01ffcae0fff0c2d1b7f) for now:

https://github.com/user-attachments/assets/b5ca36a9-d260-41aa-9c0b-2e774da2769c

☝️ this is a video showing that commas are being counted when they should be ignored.

👇 here is a video showing current behavior on prod. This shows that commas are treated purely as delimiters for adding multiple Subjects at a time, and they should not be counted during frontend validation

https://github.com/user-attachments/assets/af603037-5848-4ad1-971d-19ab333c6cda

Reply:
Fixed the issue @devincowan , please review once.